include/ibamr/IBFESurfaceMethod.h

```diff
@@ -111,7 +120,14 @@ class IBFESurfaceMethod : public IBStrategy
     static const std::string NORMAL_VELOCITY_SYSTEM_NAME;
```

Comment:
Following the suggestion below, suggest ordering variables into groups: positions; velocities; forces; jump conditions; post-processed quantities (tractions etc).

(Alternatively, please use alphabetical order.)